### PR TITLE
temp: Removed cloned button import

### PR DESCRIPTION
**Pull request by LakshananS**

### CODE REVIEW

File: server/src/components/calendar.tsx [hunk 1]
```diff
@@ -17,7 +17,6 @@ import {
   AlertDialogTitle
 } from "@/components/ui/alert-dialog";
 import { Input } from "antd";
```
Comment (bsaranga) [inline server/src/components/calendar.tsx]: Let's replace ant-design components with ShadCn.

**Review by bsaranga — CHANGES_REQUESTED** [review]

**Comment by bsaranga** [comment]
Please replace all antd usage with equivalent shadcn components. And remove the antd package.

**Comment by LakshananS** [comment]
Okay. Noted I will change all the Ant design codes.

**Comment by LakshananS** [comment]
Removed all antd package and components. Created new shadcn/ui components for input and text area. Implemented those code with frontend.

File: package.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (bsaranga) [inline package.json:1]: Let's remove this

File: package-lock.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (bsaranga) [inline package-lock.json:1]: Let's remove this

**Review by bsaranga — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (LakshananS) [reply package.json:1]: I removed those file

**Pull request merged**